Source/NameServer/SBase.cpp

```diff
@@ -0,0 +1,1550 @@
+
```

Comment:
This file is huge! Could you split it into "Constructor and OnX" commands and handlers, and Table RW methods?

Reply:
Good idea. I will split into separate files for each sub-command tree. However, I don't think this should hold up this PR, it can be done as a set of later modifications.